### DISCUSSION TITLE
How to create certs?

### DISCUSSION THREAD

**Discussion by uBoscuBo** [Azure/azure-iot-sdk-c#2345] [Q&A] [answered]
Hello, 

I am sorry if this is a trivial question; I am relatively new. I am trying to connect to Azure IoT Hub through paho-MQTT, and one of the attributes needed is a .pem and a .crt file. Through some research, I have to use the certs files, but I have no idea how to use it. I have a header and a .c file, but I can't compile it. Is there something specific I need to do?

Best,

Ayub

**Comment by danewalton-msft** [comment, marked as answer]
Hi @uBoscuBo 
We have some basic instructions on how to create those here: https://github.com/Azure/azure-sdk-for-c/tree/main/sdk/samples/iot#linux-certificate-creation

Please know that these should be used for testing only and do not serve as production grade certificates. For those, you would have to go through a Certificate Authority.
